### DISCUSSION TITLE
BLE Gateway: Support for Human presence Wave Radar Module HLK-LD2410B

### DISCUSSION THREAD

**Discussion by MDeeApp** [arendst/Tasmota#18197] [Ideas]
Hi there,

I would love to see support for LD2410B Module via Bluetooth (BLE). It is already possible to do the reading of the module by wiring it up. 
BLE support would be great in order to read multiple of the devices running by just powering them up.

Technical informations can be found here:
https://drive.google.com/drive/folders/1p4dhbEJA3YubyIjIIC7wwVsSo8x29Fq-

Home Assistant already has an integration for it:
https://www.home-assistant.io/integrations/ld2410_ble/

Thanks

**Comment by Staars** [comment]
I wrote a Berry driver some months go, but I am almost sure, that the intended purpose in the hardware design regarding Bluetooth is to only do the configuration with a Bluetooth app and not to permanently read out the sensor values. The sensor uses an active connection and thus is a bad candidate for IOT via Bluetooth.


https://gist.github.com/Staars/003c790e4b1cba9eeaf734c421131666
